tests/integration/cmd_pref_config_test.go

```diff
@@ -366,11 +366,15 @@ var _ = Describe("odo preference and config command tests", func() {
 	})
 
 	Context("Prompt should not appear when", func() {
-		It("ConsentTelemetry is set to true", func() {
-			helper.CmdShouldPass("odo", "preference", "set", "ConsentTelemetry", "true", "-f")
-			output := helper.CmdShouldPass("odo", "create", "nodejs", "--context", commonVar.Context)
-			Expect(output).ToNot(ContainSubstring(promtMessageSubString))
-		})
+
+		// This test is disable because it sends out the telemetry data and messes up the statistics
+
+		// It("ConsentTelemetry is set to true", func() {
+		// 	helper.CmdShouldPass("odo", "preference", "set", "ConsentTelemetry", "true", "-f")
+		// 	output := helper.CmdShouldPass("odo", "create", "nodejs", "--context", commonVar.Context)
+		// 	Expect(output).ToNot(ContainSubstring(promtMessageSubString))
+		// })
+
```

Comment:
Can't believe I missed something like this :facepalm: 
Instead of commenting this test, I suggest we remove it and rename the other `It` to 'ConsentTelemetry is set", it makes more sense IMO.

Reply:
I think that here it might make sense to keep it commented out. It might prevent from someone writing similar test again :)

Reply:
Comment - 
Do not test with `true` because it sends out the telemetry data and messes up the statistics.

Reply:
updated